entrypoint/selinux/selinux_test.go

```diff
@@ -0,0 +1,91 @@
+package selinux
+
+import (
```

Comment:
you should also add this test on the makefile running local and in the docker run